edgelet/edgelet-hsm/tests/crypto_create_cert_input_fail.rs

```diff
@@ -2,14 +2,25 @@
 extern crate edgelet_core;
 extern crate edgelet_hsm;
 
-use edgelet_core::{CertificateProperties, CertificateType, CreateCertificate};
+use edgelet_core::{
+    CertificateIssuer, CertificateProperties, CertificateType, CreateCertificate, IOTEDGED_CA_ALIAS,
+};
 use edgelet_hsm::Crypto;
 
 #[test]
 fn crypto_create_cert_input_fail() {
     // arrange
     let crypto = Crypto::new().unwrap();
 
+    let edgelet_ca_props = CertificateProperties::new(
+        3600,
+        "test-iotedge-cn".to_string(),
+        CertificateType::Ca,
+        IOTEDGED_CA_ALIAS.to_string(),
+    ).with_issuer(CertificateIssuer::DeviceCa);
+
+    let _workload_ca_cert = crypto.create_certificate(&edgelet_ca_props).unwrap();
```

Comment:
The other tests are expected to by virtue of invalid parameters etc. If the workload CA certificate is not created, then all calls will fail as because their issuer certificate i.e. "the workload certificate would not be found".